engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content-hosts-helper.service.js

```diff
@@ -7,6 +7,19 @@
  */
 angular.module('Bastion.content-hosts').service('ContentHostsHelper',
     function () {
+        this.convertMemToGB = function (memoryValue) {
+            if (angular.isString(memoryValue)) {
+                memoryValue = memoryValue.toLowerCase();
+                if (_.includes(memoryValue, "gb")) {
+                    memoryValue = memoryValue.replace("gb", "").trim();
+                    return memoryValue;
```

Comment:
I'd prefer one return point from this function, I feel like that's easier to follow.